app/scripts/controllers/push-platform-notifications/services/services.ts

```diff
@@ -286,5 +316,8 @@ export async function updateTriggerPushNotifications(
     notificationLinks.registration_tokens,
   );
 
-  return isTriggersLinkedToPushNotifications;
+  return {
+    isTriggersLinkedToPushNotifications,
+    fcmToken: newRegToken ?? undefined,
```

Comment:
Would `newRegToken` be `undefined` anyway if it didn't exist, or would it be `null`?

Reply:
newRegToken is null. @matteoscurati add types if they are missing.